tests/infra/ccf.py

```diff
@@ -210,10 +210,11 @@ def start_and_join(self, args):
         cmd = ["rm", "-f"] + glob("member*.pem")
         infra.proc.ccall(*cmd)
 
+        initial_members = list(range(max(1, args.initial_member_count)))
```

Comment:
Could this be `list(range(0, args.initial_member_count))`?

Reply:
We assume elsewhere that `member0` exists and we can do things as them, so I think this `max` is safer.

Reply:
Good point! This always guarantee that we've got one member.